samples/order/processor/processor.go

```diff
@@ -0,0 +1,251 @@
+// Copyright 2019 The Go Cloud Development Kit Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// A processor processes images in orders. It receives requests by
+// subscribing to the requests topic, and writes responses to the response
+// topic.
+//
+// At present, it just encodes input images into PNG format.
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"image"
+	_ "image/jpeg"
+	"image/png"
+	"log"
+	"time"
+
+	"gocloud.dev/blob"
+	_ "gocloud.dev/blob/fileblob"
+	"gocloud.dev/docstore"
+	_ "gocloud.dev/docstore/memdocstore"
+	"gocloud.dev/gcerrors"
+	"gocloud.dev/pubsub"
+	_ "gocloud.dev/pubsub/mempubsub"
+	"gocloud.dev/samples/order/internal/common"
+)
+
+var (
+	requestSubURL    = flag.String("request-sub", "", "URL for request pub/sub subscription")
+	responseTopicURL = flag.String("response-topic", "", "URL for response pub/sub topic")
```

Comment:
For each of these, I'd suggest replacing "URL for..." with "gocloud.dev/docstore URL for..." or something like that. For example, "URL for order collection" is not clear about what kind of URL it is.

Reply:
done